.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.idea
```

Comment:
I encourage you to use [VSC](https://code.visualstudio.com/) (Visual Studio Code) on front-end repos. 
In such case you would like to consider adding `.vscode` to ignore list too.